sdk/src/beta9/abstractions/endpoint.py

```diff
@@ -117,6 +117,8 @@ def __init__(
         callback_url: Optional[str] = None,
         task_policy: TaskPolicy = TaskPolicy(),
     ):
+        # Default to 1 concurrent requests for endpoints
+        self.concurrent_requests = 1
```

Comment:
do we need this? Seems like the && isASGI will just ignore this anyways

Reply:
Yeah this is actually in the wrong place, but we do need to set the default. `ASGI` calls the `Endpoint` init method and overrides the concurrent_requests with whatever it was passed. For endpoints we don't want to expose the setting in the decorator.